### ISSUE TITLE
Fix error: undefined reference to `RdKafka::Conf::create(RdKafka::Conf::ConfType)'

### ISSUE THREAD

**Issue by dmccoystephenson** [dmccoystephenson/librdkafka-testing-with-confluent-cloud#1]
## Context
This error occurs during compilation.

## Output
```
---
Compiling testing.cpp
gcc src/testing.cpp -lstdc++ -o src/testing
/usr/bin/ld: /tmp/cc2r7Hpi.o: in function `main':
testing.cpp:(.text+0xed): undefined reference to `RdKafka::Conf::create(RdKafka::Conf::ConfType)'
/usr/bin/ld: testing.cpp:(.text+0x61d): undefined reference to `RdKafka::Producer::create(RdKafka::Conf const*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&)'
/usr/bin/ld: testing.cpp:(.text+0x70b): undefined reference to `RdKafka::Topic::PARTITION_UA'
collect2: error: ld returned 1 exit status
make: *** [Makefile:14: testing] Error 1
```

## Makefile
```
all: testing
testing: src/testing.cpp
    @echo "---"
    @echo "Compiling testing.cpp"
    gcc src/testing.cpp -lstdc++ -o src/testing
    @echo "Finished compiling testing.cpp"\n
```

**Comment by dmccoystephenson** [comment]
I probably need to link librdkafka.

**Comment by dmccoystephenson** [comment]
Using cmake might be useful here.

**Comment by dmccoystephenson** [comment]
Earlier I tried:
- Compiling and installing librdkafka
- Including compiled object files from librdkafka in the compilation command.
- Including header files from librdkafka in the compilation command.
- Using docker to build a container like the ACM does. 
  https://github.com/dmccoystephenson/librdkafka-testing-with-confluent-cloud/blob/dockerfile-utilization/Dockerfile

I ended up running into more errors when trying to include the compiled objects and header files from librdkafka. I figured that I was going about this the wrong way, so I'm looking into other solutions. I'm considering using cmake.
